pkg/beholder/pb/keystone/keystone_custom.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```

Comment:
Reading the thread you linked me to yesterday, does it make sense to put this here? As opposed to in pkg/capabilities/pb?

Reply:
Is capabilities the right dir for it? A msg with some labels could be used pretty much everywhere. Mb that's more appropriate in a beholder general dir? Then other consumers can perhaps consume it and add custom fields

WDYT (@cll-gg)?

Reply:
refactored to sit in pkg/beholder/pb

345595f38e0ec17b8a5f1f488aed12aca0414d86